tzrec/main.py

```diff
@@ -336,6 +338,20 @@ def _log_train(
         summary_writer.add_scalar("loss/total", total_loss, step)
         for i, g in enumerate(param_groups):
             summary_writer.add_scalar(f"lr/g{i}", g["lr"], step)
+        for name, param in params.items():
```

Comment:
add a  summary of norm of gradient and gradient global norm like
https://github.com/alibaba/EasyRec/blob/master/easy_rec/python/compat/optimizers.py#L84 ?